lib/api/model/model.dart

```diff
@@ -569,7 +569,7 @@ sealed class Message {
   String content;
   final String contentType;
 
-  // final List<MessageEditHistory> editHistory; // TODO handle
+  final List<MessageEditHistory>? editHistory; // TODO handle
```

Comment:
This new field needs to be kept up to date in response to server events. From #171:
> The first phase of work on this issue will be to start storing and maintaining that information. This will involve adding a field to the `Message` class, and then updating that field as appropriate in response to events.